full/scripts_training/pareto_choose_point.m

```diff
@@ -27,10 +30,24 @@
 % Play with the following two number to get the desired point
 
 % Which measure used to get the Pareto
-pareto_meas = 2;  % Jaccard at class level
+prompt = "Set pareto meas [1:" + length(pareto.st) + "]: ";
+check_array = 1:length(pareto.st);
+
+pareto_meas = 0;  % Jaccard at class level
+while not(any(check_array(:) == pareto_meas) && isnumeric(pareto_meas))
+    pareto_meas = input(prompt);
+end
 
 % Which particular point in the curve we choose
-pareto_id = 64;   % Around 14k candidates
+num_masks = length(pareto.st{pareto_meas}{params.n_r_cand}.mean_n_masks);
+prompt = "Set pareto meas [1:" + num_masks + "]: ";
```

Comment:
I guess this should be "Set Pareto point"?

Reply:
Yes or Set pareto id. classical copy&pasta :)